server.py

```diff
@@ -14,33 +14,52 @@ class simulator:
     def __init__(self):
         self.parser = parser("scenarios/test.scn")
         self.scn = self.parser.parse()
+
+        # # TODO: Do this as a unit test. It's currently broken.
```

Comment:
Have a look at this - It'll be the same error as when you run server+client currently.